### PR TITLE
Added sanya.markdown

### PR DESCRIPTION
**Pull request by SanyaMehra**
Added sanya.markdown in _authors folder

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/sanya.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Sanya Mehra
+short_name: Sanya
+twitter: <twitter handle without the @>
```
Comment (ashwin-pc) [inline _authors/sanya.markdown:4]: Can you remove this line?

**Review by ashwin-pc — COMMENTED** [review]
@SanyaMehra can you move the file to `_authors/osci_2023` folder? Sorry there was a mistake in my instructions. Apologies for that.

**Comment by SanyaMehra** [comment]
Hello Ashwin,

The markdown file should be in the correct folder now, but I am not sure
which line from the file you are specifying to remove. I am also not sure
if I correctly completed all the steps because when I compare changes in
GitHub, I am unable to select my own branch in the last compare dropdown
and therefore unable to click the green create pull request button. Let me
know if I am missing something.

Thanks,
Sanya

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**